core/includes/bootstrap.inc

```diff
@@ -272,7 +272,7 @@ if (!defined('DATE_RFC7231')) {
 
 // Exit early if running an incompatible PHP version to avoid fatal errors.
 if (version_compare(PHP_VERSION, BACKDROP_MINIMUM_PHP) < 0) {
-  print 'Your PHP installation is too old. Backdrop CMS requires at least PHP 5.3.2. See the <a href="https://backdropcms.org/requirements">System Requirements</a> page for more information.';
+  print 'Your PHP installation is too old. Backdrop CMS requires at least PHP ' . BACKDROP_MINIMUM_PHP . '. See the <a href="https://backdropcms.org/requirements">System Requirements</a> page for more information.';
```

Comment:
Shouldn't we be using `t()`/`st()` here? ...in which case the concatenation of `BACKDROP_MINIMUM_PHP` in the string should go in a replacements array (as should the URL in `href`).

Reply:
...also noting that we're using the very same string in 3 places:
- bootstrap.inc
- update.php
- install.php

Perhaps worth consolidating that in a single place instead.

Reply:
These places don't use `t()` because it's so early in the bootstrap that multilingual isn't available. I think it's this way specifically because if other .inc files were included that used unsupported language features, it would immediately crash PHP rather than give a "friendly" error message.